docs/spark/tutorials/hdinsight-deployment.md

```diff
@@ -9,124 +9,180 @@ ms.custom: mvc
 
-### Deploy Microsoft.Spark.Worker
+2. Microsoft.Spark.Worker helps Apache Spark execute your app, such as any user-defined functions (UDFs) you may have written. Download [Microsoft.Spark.Worker](https://github.com/dotnet/spark/releases/download/v0.3.0/Microsoft.Spark.Worker.netcoreapp2.1.linux-x64-0.3.0.tar.gz). Then, select **Upload** in Azure Storage Explorer to upload the worker.
 
-This step is only required once for your cluster.
+   ![Upload files to Azure Storage Explorer](./media/hdinsight-deployment/upload-files-to-storage.png)
 
-Run `install-worker.sh` on the cluster using [HDInsight Script Actions](https://docs.microsoft.com/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux).
+3. The install-worker.sh is a script that lets you copy .NET for Apache Spark dependent files into the nodes of your cluster. 
 
-|Setting|Value|
-|-------|-----|
-|Script type|Custom|
-|Name|Install Microsoft.Spark.Worker|
-|Bash script URI|The URI to which you uploaded `install-worker.sh`. For example, `abfss://<your-file-system-name>@<your-storage-account-name>.dfs.core.windows.net/<some dir>/install-worker.sh`|
-|Node type(s)|Worker|
-|Parameters|Parameters to `install-worker.sh`. For example, if you uploaded `install-worker.sh` to Azure Data Lake Gen 2 then it would be `azure abfss://<your-file-system-name>@<your-storage-account-name>.dfs.core.windows.net/<some dir>/Microsoft.Spark.Worker.<release>.tar.gz /usr/local/bin`.|
+   Create a new file named **install-worker.sh** your local computer, and paste the [install-worker.sh contents located on GitHub](https://raw.githubusercontent.com/dotnet/spark/master/deployment/install-worker.sh). Then, upload install-worker.sh to your blob container.
 
-![Script Action Image](./media/hdinsight-deployment/deployment-hdi-action-script.png)
+4. Your cluster needs the publish.zip folder that contains your app's published files. Navigate to your published folder, **mySparkApp/bin/Release/netcoreapp2.2/ubuntu.16.04-x64**, and locate **publish.zip**. Then upload publish.zip to your blob container.
 
-## Run your app
+5. Your cluster needs your application code that was packaged into a jar file. Navigate to your published folder, **mySparkApp/bin/Release/netcoreapp2.2/ubuntu.16.04-x64**, and locate **microsoft-spark-2.3.x-0.3.0.jar**. Then upload your jar file to your blob container.
+
+   There may be multiple .jar files (for versions 2.3.x and 2.4.x of Spark). You need **microsoft-spark-2.3.x-0.3.0.jar** since you chose Spark 2.3.2 during cluster creation.
+
+6. Your cluster needs the input to your app. Navigate to your **mySparkApp** directory and locate **input.txt**. Upload your input file to the **user/sshuser** directory in your blob container. You will be connecting to your cluster through ssh, and this folder is where your cluster looks for its input. The input.txt file is the only file uploaded to a specific directory.
 
-You can submit your job to Azure HDInsight using `spark-submit` or Apache Livy.
```

Comment:
Users can still perform the Apache Livy steps to submit their jobs to Azure HDI. The document I was creating was intended for the in-browser tutorial format, so I just chose one option (since the in-browser tutorials try not to overload users with options). Since this is a doc instead, I believe it'd be worthwhile to keep the Apache Livy instructions in this doc.